.evergreen/config.yml

```diff
@@ -400,7 +400,8 @@ pre:
   - func: "install dependencies"
 
 post:
-  - func: "upload working dir"
+  # Skip: uploading the full working directory is not needed by drivers-evergreen-tools.
+  # - func: "upload working dir"
```

Comment:
I've never needed to download the drivers/working-dir archives. They can be very large (300+ MB on Windows) and can take multiple minutes to upload. I don't think we need this.